components/data-table/column.jsx

```diff
@@ -41,47 +41,14 @@ DataTableColumn.propTypes = {
 	 * </DataTable>
 	 * ```
 	 */
-	children: PropTypes.element,
 	/**
 	 * Some columns, such as "date last viewed" or "date recently updated," should sort descending first, since that is what the user probably wants. How often does one want to see their oldest files first in a table? If sortable and the `DataTable`'s parent has not defined the sort order, then ascending (A at the top to Z at the bottom) is the default sort order on first click.
 	 */
 	isDefaultSortDescending: PropTypes.bool,
-	/**
-	 * Selects this column as the currently sorted column.
-	 */
-	isSorted: PropTypes.bool,
-	/**
-	 * The column label. If a `string` is not passed in, no `title` attribute will be rendered.
-	 */
-	label: PropTypes.oneOfType([PropTypes.string, PropTypes.node]),
-	/**
-	 * The primary column for a row. This is almost always the first column.
-	 */
-	primaryColumn: PropTypes.bool,
-	/**
-	 * The property which corresponds to this column.
-	 */
-	property: PropTypes.string,
 	/**
 	 * Whether or not the column is sortable.
 	 */
 	sortable: PropTypes.bool,
-	/**
-	 * The current sort direction. If left out the component will track this internally. Required if `isSorted` is true.
-	 */
-	sortDirection: PropTypes.oneOf(['desc', 'asc']),
```

Comment:
This is used [here](https://github.com/salesforce/design-system-react/blob/0ee2e17692e7f5ca2ab48438d6c8558236568751/components/data-table/__examples__/advanced-single-select-fixed-header.jsx#L139). So it might not be a good idea to remove it.